MaterialDesignThemes.UITests/WPF/ComboBoxes/ComboBoxTests.cs

```diff
@@ -257,4 +257,34 @@ public async Task ComboBox_WithHintAndValidationError_RespectsPadding(string sty
 
         recorder.Success();
     }
+
+    [Theory]
+    [InlineData(HorizontalAlignment.Left, 0)]
+    [InlineData(HorizontalAlignment.Right, 1554)]   // Value obtained by inspection
+    [InlineData(HorizontalAlignment.Center, 778)]   // Value obtained by inspection
```

Comment:
The expected values here are wrong in the pipeline, because I am running DPI scaling on 150% locally. Simply doing the math of dividing by my DPI multiplier is not really sufficient, because there is also an extra UI element taking up some space (the chevron down arrow button).

Do you have some ideas on how to better assert on the `Center` and `Right` placements?

Reply:
Thanks @Keboo. I'll just clean up the no longer needed comments and enable auto merge after that.